pep-0544.txt

```diff
@@ -780,8 +780,8 @@ For example::
   class C:
       def meth(self, x: int) -> int: ...
 
-  a: ProtoA = C  # Type check error, signatures don't match!
-  b: ProtoB = C  # OK
+  a: Type[ProtoA] = C  # OK
```

Comment:
This change is incorrect. We discussed this in #2332 recently.

Reply:
Hahaha! I found this issue last week, but I didn't get around to making a PR until after #2332.

Thanks for letting me know that this is a long-outstanding bug in mypy: <https://github.com/python/mypy/issues/4536>. That issue also links to #1231, so this is at least the third attempt to fix this example. When it makes it into the typing documentation, there should probably be some improved wording.

Reply:
Also, I should have searched for this first before trying to fix it. I'm sorry for not doing so. It would have saved us both some time.